content/en/aws/feature-coverage.md

```diff
@@ -43,7 +43,7 @@ In the coverage tables below, the features are marked with their respective avai
 | Webhooks                                                           | ⭐⭐           |                 |       |
 | **API Gateway**                                                    |                |                 |       |
 | API Keys                                                           | ⭐⭐⭐         |                 |       |
-| Authorizers (Pro)                                                  | ⭐⭐⭐⭐       |                 |       |
```

Comment:
We probably want to revert that change, looks like this was a misunderstanding (still a Pro feature). Can you please confirm @calvernaz ?

Reply:
Confirm, I jumped the gun too fast